model/user.py

```diff
@@ -17,6 +17,25 @@ def __init__(self, slack_id):
         self.__image_url = ""
         self.__permissions_level = Permissions.member
 
+    def get_md_table(self):
+        """Return markdown table for user."""
+        form = """```
```

Comment:
just a thought - instead of using markdown tables for everything, Slack offers a very flexible [attachments API](https://api.slack.com/docs/message-attachments)

Reply:
That looks really nice! Will check it out.

Reply:
Like this instead?

![image](https://user-images.githubusercontent.com/6824907/49926291-17917100-fe70-11e8-9014-7ca91231b922.png)

Reply:
Hey that looks dope I love it!